src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -197,6 +134,42 @@ def __enter__(self):
         self._inputs_queue = Queue()
         self._n_process = self._determine_worker_count()
 
+        self._input_queues = [manager.Queue() for _ in range(self._n_process)]
+        self._output_queues = [manager.Queue() for _ in range(self._n_process)]
+        self._control_signal_queue = manager.Queue()
+        self._process_info = manager.dict()
```

Comment:
Maybe `process_info_dict` to indicate it is a dict to store process infos.
Currently it seems to be passing initialized process infos to subprocesses.

Reply:
Process_info is a dictionary shared between multiple processes, similar to input_queue and output_queue. 
It changes in real-time in the child process and gets the latest value in the main process.